### PR TITLE
Feat/5604 emissions api dto validation updates

### PR DESCRIPTION
**Pull request by fanwari1**
## Pull Request

Updated dto checks based on https://usepa.sharepoint.com/:x:/r/sites/CAMDCVPTeam/_layouts/15/doc2.aspx?sourcedoc=%7B742c08f5-ecf8-4a5a-b3ba-aa21fe2d579b%7D&action=edit&activeCell=%27Emissions%27!N85&wdinitialsession=20316197-f62e-cc4f-77e0-2f6fc9721607&wdrldsc=5&wdrldc=1&wdrldr=AccessTokenExpiredWarning%2CRefreshingExpiredAccessT

### CODE REVIEW

File: src/dto/daily-calibration.dto.ts [hunk 1]
```diff
@@ -42,22 +46,31 @@ export class DailyCalibrationBaseDTO {
...
   @IsOptional()
   @IsNumber()
+  @Min(0)
+  @Max(10000000000)
   upscaleMeasuredValue?: number;
 
   @IsOptional()
   @IsNumber()
+  @IsIn([0, 1])
   zeroAPSIndicator?: number;
```
Comment (sdhawan16) [inline src/dto/daily-calibration.dto.ts:74]: Does @IsIn([0,1]) need to be added for upscaleAPSIndicator as well?

File: src/dto/derived-hourly-value.dto.ts [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
-    },
-  })
+  // @IsValidCode(ModcCode, {
+  //   message: (args: ValidationArguments) => {
+  //     return ImportCodeErrorMessage(args.property, args.value);
+  //   },
+  // })
   modcCode: string;
 
   @IsString()
```
Comment (sdhawan16) [inline src/dto/derived-hourly-value.dto.ts:41]: Format validation needed for monitoringSystemId

**Review by sdhawan16 — CHANGES_REQUESTED** [review]

File: src/dto/derived-hourly-value.dto.ts (hunk 2 above)
Comment (fanwari1) [reply src/dto/derived-hourly-value.dto.ts:41]: Updated

File: src/dto/daily-calibration.dto.ts (hunk 1 above)
Comment (fanwari1) [reply src/dto/daily-calibration.dto.ts:74]: Updated

**Review by sdhawan16 — APPROVED** [review]

**Pull request merged**